silo/views.py

```diff
@@ -460,6 +460,14 @@ def edit_silo(request, id):
         tags = request.POST.getlist('tags')
         post_data = request.POST.copy()
 
+        shared_list = []
```

Comment:
This validation should be done in the form, so when we call the function `is_valid()`

Reply:
To have an idea how to do it, you can check the form `NewTolaUserRegistrationForm` from Activity, where we check the organization.